src/test/java/com/EighthLight/app/TestConfig.java

```diff
@@ -113,11 +119,21 @@ void gameMode2StartsWithAHumanAndAi() {
         MockUi ui = new MockUi(gameModeTwoUserInputs);
         Config config = new Config(ui);
 
-        ArrayList players = config.getPlayers();
+        ArrayList<IPlayer> players = config.getPlayers();
         assertTrue(players.get(0) instanceof Player);
         assertTrue(players.get(1) instanceof Ai);
     }
 
+    @Test
+    void selecting2ForPlayerOrderLetsPlayerTwoGoFirst() {
+        MockUi ui = new MockUi(playerOrderingTwoUserInputs);
+        Config config = new Config(ui);
+        ArrayList<IPlayer> players = config.getPlayers();
+
+        assertTrue(players.get(0) instanceof Ai);
+        assertTrue(players.get(1) instanceof Player);
+    }
+
```

Comment:
Should there be another test for testing player symbols like we talked about? Or are you adding that to tech debt?